utility-emissions-channel/typescript_app/tests/test.js

```diff
@@ -21,15 +22,17 @@ describe("Test fabric", function() {
       .set("content-type", "application/x-www-form-urlencoded")
       .send({ orgName: "auditor1" })
       .end((error, response) => {
-        assert((response.body.info == "An identity for the admin user already exists in the wallet") || (response.body.info == "ORG ADMIN REGISTERED"), "Failed to register auditor1.");
-        
-        if (error) 
-        {
-          done(error);
-        } else 
-        {
-          done();
+      
+       try{
+        expect(response.body.info).to.equal("ORG ADMIN REGISTERED");
+        done();
+       }
+      
+        catch(error){
+         done(error);
         }
+    
+    
       });
   });
 
```

Comment:
Currently, I have restructured the tests a bit.
If you try to register an "Auditor" or a "User" that has already been recorded previously, the test would fail. The test would only pass if we get the response as "User/Auditor successfully recorded".
Is this an ideal behaviour ?